common/pref_names.h

```diff
@@ -59,6 +59,7 @@ extern const char kBraveWalletAES256GCMSivNonce[];
 extern const char kBraveWalletEncryptedSeed[];
 extern const char kBraveWalletEnabled[];
 extern const char kAlwaysShowBookmarkBarOnNTP[];
+extern const char kTorDisabled[];
```

Comment:
IIUC, `brave/common/tor/pref_names.h` includes prefs that used in internal tor.
I think `kTorDisabled` is switch for on/off the tor feature itself.
So, I think adding it here is more appropriate. WDYT?

Reply:
@darkdh  Hmm, after some thought, I think gathering tor related prefs into one place seems fine.
and I think local state seems more better place for this new prefs.
So, I tried to add as you suggested. but there are some issues.
`BraveExtensionManagement` is created before `TorProfileService:: RegisterLocalStatePrefs()` is called.
Because of this, I can't check `kTorDisabled` in `BraveExtensionManagement::RegisterBraveExtensions` or `CleanupBraveExtensions`.
I think we need some refactoring. WDYT?

Reply:
Agree with LocalStatePrefs but I'm sure that `brave::RegisterLocalStatePrefs()` is called before `BraveExtensionManagement` being instantiated

Reply:
Moved `kTorProfile` prefs to local state. Thanks!